### ISSUE TITLE
Invalid Response

### ISSUE THREAD

**Issue by cdr-chakotay** [JavideSs/insta-extract#1]
Hi there, 

if I am attempting to scrape a user's profile, I am receiving an Invalid Request Error from the Instagram API:

```
 ___           _                    _                  _        
|_ _|_ __  ___| |_ __ _    _____  _| |_ _ __ __ _  ___| |_      
| || '_ \/ __| __/ _` |  / _ \ \/ / __| '__/ _` |/ __| __|      
| || | | \__ \ || (_| | |  __/>  <| |_| | | (_| | (__| |_  :p   
|__|_| |_|___/\__\__,_|  \___/_/\_\___|_|  \__,_|\___|\__|      

=============================================================
Traceback (most recent call last):
  File "/Users/florian/Library/Mobile Documents/com~apple~CloudDocs/Programmieren/Python/insta-extract/main.py", line 599, in <module>
    user_info_data = user_info(args["user"], args["download_posts"])
  File "/Users/florian/Library/Mobile Documents/com~apple~CloudDocs/Programmieren/Python/insta-extract/main.py", line 291, in get_user_info
    results = json.loads(response)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

```

The cause is, that the response has no payload. See image below:
<img width="763" alt="grafik" src="https://user-images.githubusercontent.com/60937022/175828933-155773e5-0f70-4db9-a454-5abe24e82063.png">

A possible fix is discussed here:
https://stackoverflow.com/a/72582413

I tested this and it works for me.

**Comment by JavideSs** [comment]
Hi,

Today I tried to use the script and I also got the same error.

I also found that adding the __d=dis worked, but after a few attempts my account was temporarily banned xd

It seems that instagram intends to change the api or make it more restrictive. The request https://i.instagram.com/api/v1/<...> returns the same results. It's also in the post you passed https://stackoverflow.com/a/72553878/13829542

Was and is impossible for me to be active for a while. When I have time I will create a new version with the best solution.

Thanks for notifying :)

**Comment by JavideSs** [comment]
- The https://i.instagram.com/api/v1/<...> option:
Doesn't seem to work. Returns a "useragent mismatch".

- The __d=dis option:
I fear the ban. I've tried it again but instagram has changed all the keys of the node.

Everything is changing. I wait for a stable API.

If someone wants to update it for now, help is welcome :)

**Comment by JavideSs** [comment]
Fixed 735cf82
Now I have problems with authentication :(

**Comment by manuchap** [comment]
Tried everything.
Turns out rebooting worked 🤣
